flag/service/service.go

```diff
@@ -1,11 +1,13 @@
 package service
 
 import (
+	"github.com/giantswarm/route53-manager/flag/service/installation"
 	"github.com/giantswarm/route53-manager/flag/service/source"
 	"github.com/giantswarm/route53-manager/flag/service/target"
 )
 
 type Service struct {
-	Source source.Source
-	Target target.Target
+	Installation installation.Config
```

Comment:
Why `Config`? Doesn't align with the others.

Reply:
fixed